### PR TITLE
version.c: mark NA patches

### PR DESCRIPTION
**Pull request by Shougo**
I have updated the version.c to 7.4.2367(last Vim 7.4 version before Vim8.0).

version.c: mark NA patches

NA

2108: test_Netbeans
2114: tiny
2116: dosinst.c
2118: Tiny
2122: FEAT_CLIPBOARD
2129, 2130: timer
2131: channel
2133: Tiny
2138: if_py_both
2140: Tiny
2145, 2150: Win32
2153: GUI test
2154: Test_commnunicate()
2155: GUI test
2156: Win32
2157: test_job
2166, 2167: small build
2168: Makefile
2176: if_mzch
2182: term.c
2186: test_timers
2191, 2192: prototype
2193: GUI
2195: vimrun
2199: GUI
2202: small
2203: normal features
2206: ifdef
2207: xpm support feature
2210: configure
2211: mouse
2214: GUI
2216: test check sign feature
2218, 2234: ifdef
2238: term.c
2243: GUI
2247: ifdef
2252, 2253: GUI
2254: if_mzch
2257: if_python
2258, 2260: channel
2261: small
2262: viminfo
2267: Windows
2270: channel
2271: Netbeans
2281: timer
2282: mch_newshellsize()
2285: proto, Makefile
2286: Filelist, Makefile
2287: channel
2288: Windows
2289: Makefile
2290: FEAT_FLOAT
2297, 2298, 2301: channel
2302: Makefile 
2304: timer
2308: Makefile
2310: timer
2316: channel
2325: Tiny
2327: channel
2332: timer
2338: small
2340: Makefile
2342: Windows
2344: channel
2345: Makefile
2351: test_Netbeans
2352: Makefile
2358: GUI
2359, 2361: timer
2363: regexp prototype
2365: Makefile, version8.txt, style
2366: bigvim.bat

Implemented

2213: EndOfBuffer

### CODE REVIEW

**Comment by jamessan** [comment]
@bfredl already merged the timer functionality over, so I think there should be some review of the timer-related patches to see which are needed.  Some of them are interactions with partials.  Partials still need to be merged, so those will likely be relevant.

**Comment by bfredl** [comment]
I will look through the timer patches. some timer patches fixes bugs that we didn't have (like stopping the timer inside the callback) or already fixed, but maybe we should add the test from those still. Some are a bit "just why?" to me, like `timer_info()`, I will port it if someone jumps and says "I want to use this in neovim".

File: src/nvim/version.c [hunk 1]
```diff
@@ -118,56 +284,56 @@ static int included_patches[] = {
...
   // 2121,
   // 2120,
   // 2119,
-  // 2118,
+  // 2118 NA
   // 2117,
-  // 2116,
+  // 2116 NA
   // 2115,
   // 2114,
```
Comment (oni-link) [inline src/nvim/version.c]: Is listed as `NA` in commit message.

File: src/nvim/version.c [hunk 2]
```diff
@@ -74,42 +74,208 @@ static char *features[] = {
...
   // 2198,
   // 2197,
   // 2196,
-  // 2195,
+  // 2195 NA
   // 2194,
-  // 2193,
-  // 2192,
-  // 2191,
+  // 2193 NA
```
Comment (oni-link) [inline src/nvim/version.c]: This is not mentioned in the commit message.

Comment (Shougo) [inline src/nvim/version.c]: Fixed.

File: src/nvim/version.c (hunk 1 above)
Comment (Shougo) [inline src/nvim/version.c]: Fixed.

**Comment by Shougo** [comment]
I have fixed it.  Thank you @oni-link !

File: src/nvim/version.c [hunk 3]
```diff
@@ -74,42 +74,208 @@ static char *features[] = {
...
+  // 2279,
+  // 2278,
+  // 2277,
+  // 2276,
+  // 2275,
+  // 2274,
+  // 2273,
+  // 2272,
+  // 2271 NA
+  // 2270 NA
```
Comment (oni-link) [inline src/nvim/version.c]: This is not mentioned in the commit message.

Comment (Shougo) [inline src/nvim/version.c]: Fixed.

**Comment by justinmk** [comment]
Merged

**Comment by Shougo** [comment]
Thank you.

**Pull request closed without merging**